exercises/practice/variable-length-quantity/variable-length-quantity.el

```diff
@@ -0,0 +1,22 @@
+;;; variable-length-quantity.el --- Variable Length Quantity (exercism)  -*- lexical-binding: t; -*-
+
+;;; Commentary:
+
+;;; Code:
+
+
+(define-error 'incomplete
+              (error "Delete this S-Expression and write your own implementation"))
```

Comment:
I think I will change to use hex, mainly because the instructions examples use hex. Thanks

Reply:
Testing for specific errors started in #400 where it made sense to distinguish  between the two errors. We did the same on Racket for that exercise even though generally we avoid testing for specific errors there because students often raise a contract violation (more idiomatic) than an error.

Here, there’s only one error so yeah we don’t need that specificity.

Reply:
should be good now I think